care_scribe/viewsets/scribe_file.py

```diff
@@ -0,0 +1,65 @@
+from django_filters import rest_framework as filters
+from rest_framework.exceptions import ValidationError
+from rest_framework.mixins import CreateModelMixin, RetrieveModelMixin, UpdateModelMixin
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.viewsets import GenericViewSet
+
+from care_scribe.models.scribe import Scribe
+from care_scribe.models.scribe_file import ScribeFile
+from care_scribe.serializers.scribe_file import (
+    ScribeFileUploadCreateSerializer,
+    ScribeFileUploadUpdateSerializer,
+)
+
+
+class FileUploadFilter(filters.FilterSet):
+    file_category = filters.CharFilter(field_name="file_category")
+
+
+def check_permissions(file_type, associating_id, user):
```

Comment:
Function duplicated, reuse.

Reply:
this validation is dependent on the FileType which is different for scribe and care